api/language/en-GB/joomla.ini

```
@@ -1000,10 +1000,10 @@ JWARNING_UNPUBLISH_MUST_SELECT="You must select at least one item to unpublish."
 JWORKFLOW="Workflow: %s"
 JWORKFLOW_ENABLED_LABEL="Enable Workflow"
 JWORKFLOW_EXECUTE_TRANSITION="Select the transition to execute on this item."
-JWORKFLOW_EXTENSION_BLACKLIST_DESCRIPTION="Disable this plugin for listed extensions."
-JWORKFLOW_EXTENSION_BLACKLIST_LABEL="Extension Blacklist"
-JWORKFLOW_EXTENSION_WHITELIST_DESCRIPTION="Activate this plugin only for listed extensions. If used all other extensions are disabled."
-JWORKFLOW_EXTENSION_WHITELIST_LABEL="Extension Whitelist"
+JWORKFLOW_EXTENSION_DISALLOW_LIST_DESCRIPTION="Disable this plugin for listed extensions."
+JWORKFLOW_EXTENSION_DISALLOW_LIST_LABEL="Extension DisallowList"
```

```suggestion
JWORKFLOW_EXTENSION_DISALLOW_LIST_LABEL="Extension Disallow List"
```
Formalise the wording to show as separate words.